lib/ansible/modules/system/filesystem.py

```diff
@@ -35,7 +35,9 @@
   fstype:
     description:
     - File System type to be created.
+    - ext4, ext4dev, ext3, ext2, xfs, btrfs are supported since the start.
```

Comment:
You should add `choices: [ext4, ext4dev, ext3, ext2, xfs, btrfs, reiserfs, lvm]` and remove this line: [description should not list the options values ](http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-block).